components/TagPane.tsx

```diff
@@ -0,0 +1,44 @@
+import { SetStateAction, Dispatch, Fragment } from 'react'
+import { useRouter } from 'next/router'
+
+import { FetchedDataSearch, LeafLoader, TagRecordCard } from './'
+import { Details, Paragraph } from '../ui'
```

Comment:
`Details` isn't used here, but I think it should be. I think that we should continue the same pattern of having 'foldable' content that we have used on other pages.

Also, without using `Details` for the list, it doesn't look like the map is rendering on mobile views

Reply:
Ok added this in